batch2/Makefile

```diff
@@ -25,3 +26,10 @@ push: build
 deploy: push
 	python3 ../ci/jinja2_render.py '{"code":{"sha":"$(shell git rev-parse --short=12 HEAD)"},"deploy":true,"batch2_image":{"image":"$(BATCH2_IMAGE)"},"default_ns":{"name":"default"},"batch_pods_ns":{"name":"batch-pods"},"batch2_database":{"user_secret_name":"sql-batch2-batch2-admin-config"},"global":{"domain":"$(DOMAIN)"}}' deployment.yaml deployment.yaml.out
 	kubectl -n default apply -f deployment.yaml.out
+
+build-worker-image:
+	-gcloud -q compute --project $(PROJECT) instances delete --zone=$(ZONE) build-batch2-worker-image
+	gcloud -q compute --project $(PROJECT) instances create --zone=$(ZONE) build-batch2-worker-image --machine-type=n1-standard-1 --network=default --network-tier=PREMIUM --metadata-from-file startup-script=build-batch2-worker-image-startup.sh --no-restart-on-failure --maintenance-policy=MIGRATE --scopes=https://www.googleapis.com/auth/cloud-platform --image=ubuntu-minimal-1804-bionic-v20191008 --image-project=ubuntu-os-cloud --boot-disk-size=10GB --boot-disk-type=pd-ssd
+	gcloud -q compute --project $(PROJECT) instances stop --zone=$(ZONE) build-batch2-worker-image
```

Comment:
Awesome. Did you make a new image batch2-worker-4? Or is that just the next one and you prepared for that?

Reply:
I see you already made it. Thanks!

Reply:
Ah, I did make it to test this, but I didn't switch to it because I was waiting for your related PR to go in first.

Reply:
Ok. I made a PR that switches to it #7233